bpfd/src/command.rs

```diff
@@ -331,6 +331,8 @@ pub(crate) struct ProgramData {
     // populated after load
     kernel_info: Option<KernelProgramInfo>,
     map_pin_path: Option<PathBuf>,
+
+    // Only populated on get or list calls.
```

Comment:
Wait is this true? We're doing all of the populating logic nested in add_program right?